docs/reference/objects/page.md

```diff
@@ -0,0 +1,30 @@
+---
+layout: default
+title: Page
+nav_order: 2
```

Comment:
I actually meant to go through and delete all of these, we shouldn't need to specify then it'll do alphabetical